loader/src/ign_TEST.cc

```diff
@@ -183,3 +185,37 @@ TEST(ignTest, PluginInfoVerboseDummyPlugins)
             output.find("There are 2 aliases with a name collision"))
       << output;
 }
+
+//////////////////////////////////////////////////
+/// \brief Check --help message and bash completion script for consistent flags
+TEST(ignTest, PluginHelpVsCompletionFlags)
+{
+  // Path to ign executable
+  std::string ign = std::string(IGN_PATH);
+
+  // Flags in help message
+  std::string helpOutput = custom_exec_str(ign + " plugin --help");
+
+  // Call the output function in the bash completion script
+  std::filesystem::path scriptPath = IGN_PLUGIN_SOURCE_DIR;
```

Comment:
CI is unhappy, we can't use `std::filesystem` in this branch without some updates to the CMake code.

Reply:
Yay it's fixed in 9513232! Also found a bug in 47c159a